gamemodes/attack_of_the_mimics/entities/weapons/swep_aotm_claws.lua

```diff
@@ -117,7 +122,7 @@ function SWEP:PrimaryAttack()
 
 	-- effects
 	if tr.Hit then
-		self:EmitSound( self.Primary.HitSound[math.random(#self.Primary.HitSound)] )
+		self:EmitSound( table.Random(self.Primary.HitSound) )
```

Comment:
This was mentioned in the wiki to be less efficient. https://wiki.garrysmod.com/page/table/Random